### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR updates README. Moves info from AM doc 1629 to here.

**Review:**
- [ ] code approved by
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [x] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,130 @@
...
+
+
+## Branching model
+
 clinical_EPPs is using github flow branching model as described in our development manual.
+
+
+## Production and Stage
+
+Development of new EPPs is preferably be done locally but the final testing is done on the stage server.
```
Comment (henrikstranneheim) [inline README.md:16]:
```suggestion
Development of new EPPs is preferably done locally but the final testing is done on the stage server.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,130 @@
...
+python setup.py install
+
+```
+the branch that has been installed is now avalibe from within the [lims web interface](https://clinical-lims-stage.scilifelab.se/clarity/).
+
+
+
+## Setting up a new EPP
+
+The branch with the new script has been installed and you want ot test the script through the web interface. (Or deploy it on production. The procedure is the same.)
```
Comment (henrikstranneheim) [inline README.md:48]:
```suggestion
The branch with the new script has been installed and you want to test the script through the web interface. (Or deploy it to production. The procedure is the same.)
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,130 @@
...
+
+Say the new script is bcl2fastq.py. Running it from the commandline looks like this:
```
Comment (henrikstranneheim) [inline README.md:50]:
```suggestion
Let us call the new script we want to test: `bcl2fastq.py`. Running it from the command line looks like this:
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,130 @@
...
+usage: bcl2fastq.py [-h] [-p PID] [-l LOG]
+
+optional arguments:
+  -h, --help  show this help message and exit
+  -p PID      Lims id for current Process
+  -l LOG      File name for standard log file, for runtime information and
+              problems.
+```
+
+To make it avalible in the [web interface](https://clinical-lims-stage.scilifelab.se/clarity), go to the CONFIGURATON tab and then select  AUTOMATION. Klick the NEW AUTOMATON button.
```
Comment (henrikstranneheim) [inline README.md:64]:
```suggestion
To make it avalible in the [web interface](https://clinical-lims-stage.scilifelab.se/clarity), go to the `CONFIGURATON` tab and then select `AUTOMATION`. Klick the `NEW AUTOMATON` button.
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,130 @@
...
+
+- Choose a Automation Name
+- Channel Name should always be `limsserver`.
+- Enter the command line string. If you need help selecting a token for an argument, klick the TOKENS tab. In this case the string is
+`bash -c "/home/glsai/miniconda2/envs/epp_master/bin/bcl2fastq.py -p {processLuid} -l {compoundOutputFileLuid0}"`
```
Comment (henrikstranneheim) [inline README.md:69]: Are {processLuid} just place holder or actual parameters that exits in the env on the servers?
Same with this one: {compoundOutputFileLuid0}

Comment (mayabrandi) [reply README.md:69]: They are parameters that exist. Refered to as TOKENS. They can eg point to the process id to a file placeholder of the process

Comment (henrikstranneheim) [reply README.md:69]: Could you add a line or two on that because to me it was not obvious that would exist and how they are used

Comment (mayabrandi) [reply README.md:71]: Absolutely. I added a description

**Pull request merged**